tests/core/arch-arm64/arm64stalker.c

```diff
@@ -0,0 +1,652 @@
+/*
+ * Copyright (C) 2009-2014 Ole André Vadla Ravnås <ole.andre.ravnas@tillitech.com>
+ * Copyright (C) 2010-2013 Karl Trygve Kalleberg <karltk@boblycat.org>
+ *
+ * Licence: wxWindows Library Licence, Version 3.1
+ */
+
+#include "arm64stalker-fixture.c"
```

Comment:
Actually for these I followed `arm64relocator.c` `arm64relocator-fixture.c` `arm64writer.c` and `arm64writer-fixture.c` of the same folder. Confirm you wanted them the other way.